packages/vanilla/src/components/modal/modal.md

```diff
@@ -0,0 +1,39 @@
+# Modal (Dialog)
+
+### Purpose
+
+- A Modal disables page content and focuses the user’s attention on a single task or message. (*Only the content in the modal can be interacted with.*)
+
+- Modal vs. Dialog - The key difference between a modal and a dialog is that modals will render the background `inert` meaning no elements "behind" the modal are interactive. 
+
+- *Non-modal dialogs allow interaction with content outside of the dialog*
+
+### Functional Requirements
+
```

Comment:
These are very granular for functional requirements. I'd move most of this down to the technical specifications. Functional requirements should be high-level in plain language.